### PR TITLE
SE-1150: Create HelloWorld.cs

### PR DESCRIPTION
**Pull request by mneedham**
# Pull Request Checklist

- [x] Read the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Hello World Example to use in the Quick Start Guide.

### CODE REVIEW

**Review by thomasdotred — CHANGES_REQUESTED** [review]
Should there be a test for this? for instance if the upsert instruments endpoint get changed, what will alert is that this will need fixing?

**Comment by mneedham** [comment]
> Should there be a test for this? for instance if the upsert instruments endpoint get changed, what will alert is that this will need fixing?

makes sense. I've added a test that calls the example

**Review by thomasdotred — APPROVED** [review]

File: sdk/Examples/HelloWorld.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            var apiFactory = LusidApiFactoryBuilder.Build(secretsFile);
+            // end::create-client-factory[]
+
+            // tag::create-tx-portfolios-api[]
+            var txPortoliosApi = apiFactory.Api<ITransactionPortfoliosApi>();
+            // end::create-tx-portfolios-api[]
+
+            // tag::create-portfolio[]
+            var scope = "GettingStartedScope";
+            var uuid = Guid.NewGuid().ToString();
```
Comment (thomasdotred) [inline sdk/Examples/HelloWorld.cs:30]: Can you use this uuid to delete the portfolio after this is run?

File: sdk/Examples/HelloWorld.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+            // tag::get-holdings[]
+            var holdingsResponse = txPortoliosApi.GetHoldings(scope: scope, code: portfolioCode,
+                propertyKeys: new List<String>() {"Instrument/default/Name"});
+            foreach (var holding in holdingsResponse.Values)
+            {
+                Console.WriteLine($"{luidToName[holding.InstrumentUid]}, {holding.Units}, {holding.Cost.Amount}");
+            }
+            // end::get-holdings[]
+
```
Comment (thomasdotred) [inline sdk/Examples/HelloWorld.cs:95]: you could tear down this portfolio here, or pass out the scope and code to tear it down in the `HelloWorldTest.cs`

**Review by thomasdotred — CHANGES_REQUESTED** [review]

**Pull request merged**